examples/rpl-border-router/Makefile

```diff
@@ -13,9 +13,17 @@ else
 endif
 
 include $(SOURCES_DIR)/Makefile
+# Enable Web server by default
+WITH_WEBSERVER ?= 1
 
 PROJECTDIRS += $(SOURCES_DIR)
-PROJECT_SOURCEFILES += httpd-simple.c webserver.c border-router-common.c
+
+ifeq ($(WITH_WEBSERVER),WITH_WEBSERVER)
+httpd-simple.c webserver.c
+endif
```

Comment:
There is no reason to compile the web server conditionally

Reply:
(that I can immediately think of, at least...)

Reply:
The reason was https://github.com/contiki-ng/contiki-ng/blob/develop/examples/rpl-border-router/common/httpd-simple.c#L46

But maybe we just remove this check so we can configure everything from headers again?